nucypher/network/retrieval.py

```diff
@@ -143,10 +141,20 @@ class RetrievalWorkOrder:
     A work order issued by a retrieval plan to request reencryption from an Ursula
     """
 
-    def __init__(self, ursula_address: ChecksumAddress, capsules: List[Capsule], lingos: Optional[List[ConditionLingo]] = None):
+    def __init__(self,
+                 ursula_address: ChecksumAddress, capsules: List[Capsule],
+                 lingo: Optional[List[ConditionLingo]] = None
```

Comment:
should we still call this 'lingos' i.e. plural since it is a list of lingo objects? Same for the member variable.